src/Files.App/UserControls/SideBar/SideBarView.xaml

```diff
@@ -124,21 +122,15 @@
 				UseSystemFocusVisuals="True" />
 		</Border>
 
-		<Grid
+		<ContentPresenter
 			x:Name="ContentPresenter"
 			Grid.Row="1"
 			Grid.Column="2"
 			Margin="2,0,8,0"
-			HorizontalAlignment="Stretch">
-
-			<!--  Content  -->
-			<Border
-				x:Name="Content"
```

Comment:
Are there any visual states connected to this?

Reply:
This isn't defined as x:Modifier="public" so I searched in this view but no occurrences found.
And others as well.